File: Projeto_7/src/LogSeguranca.cpp
#include "LogSeguranca.h"

LogSeguranca::LogSeguranca(std::string timestamp, std::string nivel, std::string mensagem) 
    : timestamp(timestamp), nivel(nivel), mensagem(mensagem) {}

std::string LogSeguranca::getTimestamp() const { return timestamp; }
std::string LogSeguranca::getNivel() const { return nivel; }
std::string LogSeguranca::getMensagem() const { return mensagem; }

File: Projeto_7/src/LogSeguranca.py
class LogSeguranca:
    def __init__(self, timestamp: str, nivel: str, mensagem: str):
        self.__timestamp = timestamp
        self.__nivel = nivel
        self.__mensagem = mensagem

    def get_nivel(self) -> str:
        return self.__nivel

    def __str__(self) -> str:
        return f"Log [{self.__timestamp}] {self.__nivel}: {self.__mensagem}"